### ISSUE TITLE
fix factors

### ISSUE THREAD

**Issue by jhmarcus** [StoreyLab/lfa#4]
Hello,

I hope is all is well! Is it possible in the `lfa` package to fix a few factors for all individuals and estimate the remaining factors? 

Thanks, 

Joe

**Comment by whao89** [comment]
Hi Joe, do you mean that you want to include adjustment variables when computing logistic factors?

**Comment by jhmarcus** [comment]
Thanks for the quick reply! Using the notation  in model 2 / equation 2 (Hao et al. 2016) where L = AH I mean fixing some of the rows of H to known values while still estimating the other rows as well as A? What do you mean by adjustment variables?

**Comment by whao89** [comment]
Adjustment variables in this case are just additional information about individuals, like how you might control for things in a regression. As a concrete example, perhaps something like self-reported ancestry? I think this is the same as what you're suggesting. This is definitely doable, but the change has to enter the algorithm at multiple points. If you're not in a rush, I can work on and test this in a branch and get back to you.

**Comment by jhmarcus** [comment]
Yep that is what I mean. That would be great! Thank you for the help.

**Comment by whao89** [comment]
I think the `devel` branch should do what you want. The `lfa` function now has a `adjustments` argument which takes a matrix. The number of factors `d` needs to include the number of adjustment variables. 

I'm not merging into `master` yet since I reorganized a bunch of stuff and want to make sure it all works.

Let me know if anything behaves weirdly or if this doesn't work the way you expect!
